### PR TITLE
Resources-first-commit

### PR DESCRIPTION
**Pull request by bmushtuk**
Please review my first commit within the Resources branch

### CODE REVIEW

**Comment by tsukuros** [comment]
@bmushtuk Please deploy your latest changes to live site, so we can review your work.

File: index.html [hunk 1]
```diff
@@ -187,15 +187,15 @@ <h4 class="text-proxima text-20 mb-3">
               <p>This is a greate resource to get you started on project managment and stuff. This is a greate resource to get you started on project managment and stuff.</p>
             </div>
           </div>
-    
           <a href="" class="btn btn-lg btn-outline-danger mx-auto read-more-btn btn-block">
             <span class="mr-1">Get More Resources</span>
-            <svg xmlns="http://www.w3.org/2000/svg" width="19" height="12" viewBox="0 0 19 12">
+            <svg xmlns="http://www.w3.org/2000/svg" width="19" height="12" viewbox="0 0 19 12">
```
Comment (tsukuros) [inline index.html]: Why did you have to do this change?

Comment (bmushtuk) [reply index.html]: It was marked in VS Code as an error for some reason

**Comment by bmushtuk** [comment]
So, at this point I have to commit all changes (from all sections) to gh-pages first?

**Comment by tsukuros** [comment]
Yes, please merge the ones that you want me to review. In your case, it looks like `Resources`, `Blog` and `Footer`.

By the way, try not use capitalization in branch names. The convention is lowercased text with dashes.
capitalization is error-prone.

**Review by ghost — APPROVED** [review]

**Review by tsukuros — APPROVED** [review]

**Pull request merged**